src/Files.App/ViewModels/UserControls/InfoPaneViewModel.cs

```diff
@@ -136,12 +138,27 @@ private async void ContentPageContext_PropertyChanged(object? sender, PropertyCh
 
 					SelectedItem = tempSelectedItem;
 
-					if (!App.AppModel.IsMainWindowClosed)
+					try
 					{
-						var shouldUpdatePreview = ((MainWindow.Instance.Content as Frame)?.Content as MainPage)?.ViewModel.ShouldPreviewPaneBeActive;
-						if (shouldUpdatePreview == true)
-							_ = UpdateSelectedItemPreviewAsync();
+						if (!App.AppModel.IsMainWindowClosed)
+						{
+							var shouldUpdatePreview = ((MainWindow.Instance.Content as Frame)?.Content as MainPage)?.ViewModel.ShouldPreviewPaneBeActive;
+							if (shouldUpdatePreview == true)
+								_ = UpdateSelectedItemPreviewAsync();
+						}
 					}
+					catch (Exception ex)
+					{
+						// Handle exception in case WinUI Windows is closed
+						// (see https://github.com/files-community/Files/issues/15599)
+
+						SentrySdk.CaptureException(ex, scope =>
+						{
+							scope.User.Id = generalSettingsService.UserId;
+							scope.Level = SentryLevel.Warning;
+						});
+					}
```

Comment:
Shouldn't it also log to a file?
I'd like to incorporate sending reports to Sentry into the logger soon.

Reply:
I suppose